sceptre/stack.py

```diff
@@ -116,7 +116,12 @@ class Stack(object):
     parameters = ResolvableContainerProperty("parameters")
     sceptre_user_data = ResolvableContainerProperty("sceptre_user_data")
     notifications = ResolvableContainerProperty("notifications")
-    tags = ResolvableContainerProperty('tags')
+    tags = ResolvableContainerProperty("tags")
+
+    s3_details = ResolvableContainerProperty("s3_details")
```

Comment:
why is this a resolvable property?  it's not a key in stack configs is it?

Reply:
So the template_bucket_name is accessed off of this property in the template class. So, in order to make template_bucket_name resolvable in all cases, the s3_details needed to be resolvable too.